ui/src/app/pages/login/login.component.html

```diff
@@ -15,5 +15,13 @@
   -->
 
 <div class="login-form-container">
-  <app-login-form class="login-form-item"></app-login-form>
+  <app-spinner *ngIf="(isPending$ | async); else appLoginForm"></app-spinner>
+
+  <ng-template #appLoginForm >
+    <app-login-form class="login-form-item"></app-login-form>
+  </ng-template>
+
+  <div class="demo-page-link" *ngIf="(isDemoPageAvailable$ | async) && !(isPending$ | async)">
```

Comment:
remove  && !(isPending$ | async). can be moved inside <ng-template #appLoginForm > which will be rendered only if !isPending